### PR TITLE
Feature/send raw data

### PR DESCRIPTION
**Pull request by adieyal**
## Checklist

- [ ]  🚀 is the code ready to be merged and go live?
- [ ]  🛠 does it work (build) locally

### Pull Request

- [ ]  📰 good title
- [ ]  📝good description
- [ ]  🔖 issue linked
- [ ]  📖 changelog filled out

### Commits

- [ ]  commits are clean
- [ ]  commit messages are clean

### Code Quality

- [ ]  🚧 no commented out code
- [ ]  🖨 no unnecessary logging
- [ ]  🎱 no magic numbers
- [ ]  black was run locally (as part of the pre-commit hook)

### Testing

- [ ]  ✅ added (appropriate) unit tests
- [ ]  💢 edge cases in tests were considered
- [ ]  ✅ ran tests locally & are passing

### CODE REVIEW

File: tests/profile/serializers/test_metrics_serializer.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
+import pytest
+from unittest.mock import patch
+from wazimap_ng.profile.serializers.metrics_serializer import absolute_value, subindicator, sibling
+from tests.datasets.factories import IndicatorDataFactory
+
+@pytest.fixture
+def indicatordata_json():
```
Comment (milafrerichs) [inline tests/profile/serializers/test_metrics_serializer.py:7]: this again is the same as in the `conftest` right?

**Review by milafrerichs — CHANGES_REQUESTED** [review]
some minor cleanup would be great.

File: tests/datasets/tasks/indicator_data_extraction/test_extraction.py [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+        indicator_data_items = indicator_data_extraction(indicator)
+        assert len(indicator_data_items) == 1
+        assert IndicatorData.objects.count() == 1
+
+        indicator_data_items = indicator_data_extraction(indicator)
+        assert len(indicator_data_items) == 1
+        assert IndicatorData.objects.count() == 1
+
+
+    def test_three_geographies(self, indicator, datasetdata2, geographies):
```
[the remark this answers is not in the document]
Comment (adieyal) [reply tests/datasets/tasks/indicator_data_extraction/test_extraction.py]: Fixed - removed the datasetdata2 fixture altogether in favour of the child datatsets defined in conftest

File: tests/profile/serializers/test_highlights_serializer.py [hunk 3]
```diff
@@ -0,0 +1,63 @@
+import pytest
+from unittest.mock import patch
+from wazimap_ng.profile.serializers.highlights_serializer import absolute_value, subindicator, sibling
+from tests.datasets.factories import IndicatorDataFactory
+
+@pytest.fixture
+def indicatordata_json():
```
[the remark this answers is not in the document]
Comment (adieyal) [reply tests/profile/serializers/test_highlights_serializer.py]: 3612794e59c5e490b2950733214ffe2fb615935d addresses this

File: tests/profile/serializers/test_highlights_serializer.py [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+        {"gender": "female", "age": "15", "language": "isiXhosa", "count": 7},
+        {"gender": "female", "age": "15", "language": "isiZulu", "count": 8},
+        {"gender": "female", "age": "16", "language": "isiXhosa", "count": 9},
+        {"gender": "female", "age": "16", "language": "isiZulu", "count": 10},
+        {"gender": "female", "age": "17", "language": "isiXhosa", "count": 11},
+        {"gender": "female", "age": "17", "language": "isiZulu", "count": 12},
+    ]
+
+@pytest.fixture
+def indicatordata_json2(indicatordata_json):
```
[the remark this answers is not in the document]
Comment (adieyal) [reply tests/profile/serializers/test_highlights_serializer.py]: 3612794e59c5e490b2950733214ffe2fb615935d addresses this

File: tests/profile/serializers/test_highlights_serializer.py [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+
+    actual_value = subindicator(profile_highlight, geography)
+    
+    assert expected_value == actual_value
+
+    
+@pytest.mark.django_db
+def test_sibling(profile_highlight, geography, geographies):
+    with patch.object(geography, "get_siblings", side_effect=lambda: geographies[0:2]):
+        expected_value = 0.3333
```
[the remark this answers is not in the document]
Comment (adieyal) [reply tests/profile/serializers/test_highlights_serializer.py]: addressed here: 05da0ca16dd1a75c73d871cd5a550eb627254ac7

File: tests/profile/serializers/test_metrics_serializer.py (hunk 1 above)
Comment (adieyal) [reply tests/profile/serializers/test_metrics_serializer.py]: Fixed here: 3612794e59c5e490b2950733214ffe2fb615935d

File: tests/conftest.py [hunk 6]
```diff
@@ -0,0 +1,185 @@
...
+def child_geographies(geography):
+    return [
+        geography.add_child(code=f"child{i}_geo", version=geography.version)
+        for i in range(2)
+    ]
+
+
+@pytest.fixture
+def profile(geography_hierarchy):
+    _profile = ProfileFactory(geography_hierarchy=geography_hierarchy)
```
[the remark this answers is not in the document]
Comment (adieyal) [reply tests/conftest.py]: I opted for a more straightforward approach: b9ff775df755fa0ad000310611b4548c6d80a115

**Review by milafrerichs — APPROVED** [review]

**Pull request merged**